src/pages/developers/learning-tools.tsx

```diff
@@ -228,7 +228,7 @@ const LearningToolsPage = ({
     {
       name: "Platzi",
       description: "page-learning-tools-platzi-description",
-      url: "https://platzi.com/blockchain",
+      url: "https://platzi.com/startupsweb3/",
```

Comment:
```suggestion
      url: "https://platzi.com/escuela/escuela-blockchain/",
```

Reply:
Agree with @pettinarip and @minimalsm. Will commit their suggestion and pull in.